polkadot-protocol/protocol-components/transactions/transactions-and-block-basis.md

```diff
@@ -0,0 +1,69 @@
+
+### Inherent Transactions
+
+Inherent transactions, sometimes referred to as inherent, are a particular type of unsigned transaction. With this type of transaction, block authoring nodes can add information directly to a block. Inherent transactions can only be inserted into a block by the block authoring node that calls them. Typically, this type of transaction is not gossiped to other nodes or stored in the transaction queue. The data inserted using an inherent transaction is assumed valid without requiring specific validation.
+
+For example, if a block authoring node inserts a timestamp into a block, there is no way to prove that a timestamp is accurate. Instead, validators might accept or reject the block based on whether the timestamp is within some acceptable range of their system clocks.
+
+As an example, the [`pallet_timestamp::Call::now`](https://github.com/paritytech/polkadot-sdk/blob/master/substrate/frame/timestamp/src/lib.rs){target=\_blank} extrinsic enables a block authoring node to insert a current timestamp in each block the node produces. Similarly, the [`paras_inherent::Call::enter`](https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/runtime/parachains/src/paras_inherent/mod.rs#L247){target=\_blank} extrinsic enables a parachain collator node to send its relay chain the validation data the relay chain expects.
+
+## What is a Block?
+
+In Substrate, a block consists of a header and an array of transactions. The header contains the following properties:
+
+- Block height - the number of blocks that have been created in the chain
+- Parent hash - the hash of the previous block in the chain
+- Transaction root - a cryptographic digest of the transactions in the block
+- State root - a cryptographic digest of the state after executing the transactions
+- Digest - a list of additional information that can be included in the block
+
+All of the transactions are bundled together as a series to be executed as defined in the runtime. You'll learn more about transaction ordering in the [Transaction Lifecycle](TODO:update-path){target=\_blank} documentation. The transaction root is a cryptographic digest of this series. This cryptographic digest serves two purposes:
+
+- It prevents any alterations to the series of transactions after the header has been built and distributed
+- It enables light clients to succinctly verify that any given transaction exists in a block given only knowledge of the header
+
+## Where to Go Next
```

Comment:
I wouldn't say I like where to go next sections, but in this case, it seems useful. Feel free to decide about this